prometheus/base/prometheus.alerting.rules.yaml

```diff
@@ -0,0 +1,19 @@
+apiVersion: monitoring.coreos.com/v1
+kind: PrometheusRule
+metadata:
+  labels:
+    openshift.io/prometheus-rule-evaluation-scope: leaf-prometheus
+  name: o11y-alerting-rules
+  namespace: o11y
+spec:
+  groups:
+  - name: o11y_alerts
+    rules:
+    - alert: UnschedulablePods
+      expr: last_over_time(kube_pod_status_unschedulable{namespace!~".*-tenant"}[5m]) == 1
+      for: 30m
+      labels:
+        severity: warning
+      annotations:
+        summary: "Unschedulable pods"
+        description: "Alerts when pod gets stuck in 'Pending' status"
```

Comment:
The summary and description are user facing, meaning a person will receive an alert and this information is the first hint about what went wrong. Looking at the `KubePodNotScheduled` alerting rule, it has more detail than this.

```yaml
summary: Pod cannot be scheduled.
description: "Pod {{ $labels.namespace }}/{{ $labels.pod }} cannot be scheduled for more than 30 minutes. Check the details of the pod with the following command: oc describe -n {{ $labels.namespace }} pod {{ $labels.pod }}"
```

We should elaborate on the potential root cause and use variables to assist with the triage.